doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -406,11 +406,11 @@ \subsubsection{Outputs}\label{outputs-017}
 
 \paragraph{People Air Temperature {[}C{]}}\label{people-air-temperature-c}
 
-This output variable represents the zone air temperature based on the Fanger thermal comfort model.~ If there is a ZoneControl:Thermostat:ThermalComfort object specified and the thermal zone is occupied, then the value of ``People Air Temperature'' is determined based on the thermal comfort hat satisfies the thermal comfort setpoint PMV value specified; othwesie, it is set to average zone air temperature.
```

Comment:
This was by far my favorite typo.  **othwesie**  !!! And it's repeated below, of course 💯 

( @jasondegraw @jmarrec @mjwitte )